### DISCUSSION TITLE
[#1037164] Allow Brush Reach Distance to be Similar to Render Distance Setting

### DISCUSSION THREAD

**Discussion by Estrongel** [Mojang/minecraft-editor#23] [Feedback]
A slider to the editor mode that allows users to modify the distance their brush can reach. It can function similarly to how render distances function in that it takes into account a user's system to determine the max range to prevent them from selecting a distance that their system would be unable to handle.

**Comment by minecraftchloe** [comment]
Yes! This is something we absolutely need to improve on and have ongoing discussions about! This is a great idea on implementation, too!
